fdbserver/LogSystem.h

```diff
@@ -1087,13 +1088,30 @@ struct LogPushData : NonCopyable {
 		next_message_tags.clear();
 	}
 
-	Standalone<StringRef> getMessages(int loc) { return messagesWriter[loc].toValue(); }
+	Standalone<StringRef> getMessages(int loc) {
```

Comment:
instead of growing this function consider creating a new one `recordEmptyMessage(StandAlone<StringRef> value)` and call it right after the call to `getMessages()`

Reply:
Done.